build: Cannot open module file 'pmpi_base.mod'

These are the steps I have followed trying to build mpich v4.1.2

1.	fetched mpich from https://github.com/pmodels/mpich
2.	i.e., git clone https://github.com/pmodels/mpich.git
3.	cd mpich
4.	./autogen.sh to create ‘configure” file
5.	ERROR indicates need to run: git submodule update –init
6.	./configure creates the Makefile

running "make"  yields the error seen below: 

```
Running “make”  MOD      src/binding/fortran/use_mpi/mpi.mod-stamp
src/binding/fortran/use_mpi/mpi.f90:8:9:

    8 |     USE PMPI_BASE
      |         1
Fatal Error: Cannot open module file 'pmpi_base.mod' for 
reading at (1): No such file or directory
compilation terminated.
1.	make[2]: *** [Makefile:22297: src/binding/fortran/use_mpi/mpi.mod-stamp] Error 1
```

If I use the --disable-fortran flag in the configure step, it compiles, however, I need the fortran component so that is not an option.  Here are the particulars for my system (i.e., output of "uname -a"):
```
Linux debian 6.1.0-12-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.52-1 (2023-09-07) x86_64 GNU/Linux
```

"gfortran -v" shows this: 
```
gcc version 12.2.0 (Debian 12.2.0-14)
```

I saw in the "issues" a reference to this error seemingly being fixed.  Should I be pulling a different branch?

--- a/src/binding/fortran/use_mpi/Makefile.mk
+++ b/src/binding/fortran/use_mpi/Makefile.mk
@@ -68,7 +68,7 @@ mpi_fc_sources += \
 # We need the MPI constants in a separate module for some of the
 # interface definitions (the ones that need MPI_ADDRESS_KIND or
 # MPI_OFFSET_KIND)
-src/binding/fortran/use_mpi/mpi.$(MOD)-stamp: src/binding/fortran/use_mpi/$(MPICONSTMOD).$(MOD) src/binding/fortran/use_mpi/$(MPISIZEOFMOD).$(MOD) src/binding/fortran/use_mpi/$(MPIBASEMOD).$(MOD) $(srcdir)/src/binding/fortran/use_mpi/mpi.f90 src/binding/fortran/use_mpi/mpifnoext.h
+src/binding/fortran/use_mpi/mpi.$(MOD)-stamp: src/binding/fortran/use_mpi/$(MPICONSTMOD).$(MOD) src/binding/fortran/use_mpi/$(MPISIZEOFMOD).$(MOD) src/binding/fortran/use_mpi/$(MPIBASEMOD).$(MOD) src/binding/fortran/use_mpi/$(PMPIBASEMOD).$(MOD) $(srcdir)/src/binding/fortran/use_mpi/mpi.f90 src/binding/fortran/use_mpi/mpifnoext.h
 	@rm -f src/binding/fortran/use_mpi/mpi-tmp
 	@touch src/binding/fortran/use_mpi/mpi-tmp
 	@( cd src/binding/fortran/use_mpi && \

